documentation/user_docs/docs/manual/Advanced_Multifit.rst

```diff
@@ -2,133 +2,199 @@
 Advanced Multifit
 #################
 
-Parallel fitting
-================
+Multifit methods requirements
+=============================
 
-It is possible to use ``multifit`` and its derivatives (``tobyfit``, ``multifit_sqw``, ``multifit_sqw_sqw``) in parallel
-(see :ref:`manual/Parallel:Running Horace in Parallel` for more info) by either enabling HPC options through
+This section describes how to define a class which is compatible with
```

Comment:
As discussed this is not meaningful as it stands. you could either provide a minimum context to make sense of it, or you could say this is not intended to be usable but it a stored old document in case it needs to be revived by the developers. But you need to say something. No further comments for this file.